newIDE/app/src/Utils/GDevelopServices/Extension.js

```diff
@@ -5,6 +5,7 @@ import semverSatisfies from 'semver/functions/satisfies';
 
 export type ExtensionShortHeader = {|
   shortDescription: string,
+  authors?: Array<{ id: string, username: string }>,
```

Comment:
Let's go ahead and have a `Array<UserPublicProfile>` with `UserPublicProfile` in a User.js file. I know it's maybe thinking too much right now, but this will ensure the consistency of the user profile information everywhere and this will help finding all the places in the codebase using it when we add new attributes :) (avatar, color, badges...)